pkg/ebpf/c/tracee.bpf.c

```diff
@@ -8059,12 +8092,18 @@ CGROUP_SKB_HANDLE_FUNCTION(proto_tcp_http)
 {
     // submit HTTP base event if needed (full packet)
 
-    if (should_submit_net_event(neteventctx, SUB_NET_PACKET_HTTP)) {
+    if (should_submit_net_event(neteventctx, SUB_NET_PACKET_HTTP))
         cgroup_skb_submit_event(ctx, neteventctx, NET_PACKET_HTTP, FULL);
-        cgroup_skb_capture(); // capture http packets
-    }
+
+    if (should_submit_net_event(neteventctx, SUB_NET_PACKET_IP) ||
+        should_submit_net_event(neteventctx, SUB_NET_PACKET_TCP) ||
+        should_submit_net_event(neteventctx, SUB_NET_PACKET_HTTP))
+        cgroup_skb_capture(); // capture http-tcp, tcp or ip packets
```

Comment:
ditto.

```suggestion
        cgroup_skb_capture(); // capture ip, tcp or http-tcp packets
```

Reply:
This is in reverse order because that is how it should be going backwards (in the logic of capturing).